tests/fail/stacked_borrows/exposed_only_ro.rs

```diff
@@ -0,0 +1,12 @@
+// compile-flags: -Zmiri-permissive-provenance
+#![feature(strict_provenance)]
+
+// If we have only exposed read-only pointers, doing a write through a wildcard ptr should fail.
```

Comment:
This comment doesn't line up with the diagnostic. If this comment is correct, the diagnostic should say something like "but no tags for this location with write permission were exposed". Because a user could complain "This error message says no exposed tags are valid, but look I exposed a tag right here by calling `expose_addr` and I never invalidated it".

Reply:
Ah, fair. We can probably make a more targeted diagnostic but for now I'll just adjust the error text.